packages/tabs/src/Tabs.ts

```diff
@@ -101,6 +104,9 @@ export class Tabs extends SizedMixin(Focusable) {
     @property()
     public label = '';
 
+    @property({ type: Boolean, reflect: true })
```

Comment:
It doesn't look like we are styling or querying anything off of this attribute, does it need to reflect?